src/slash/database.ts

```diff
@@ -90,126 +86,146 @@ export async function authenticatePlayer(
 	return player;
 }
 
-export function generateDeckValidateButtons(): ActionRowBuilder<ButtonBuilder> {
+export function encodeCustomId(idName: string, ...args: Array<string | number>): string {
+	args.unshift(idName);
+	return args.join("#");
+}
+
+export function decodeCustomId(id: string): string[] {
+	const args = id.split("#");
+	// the result of spllit has at least 1 entry unless the string is empty
```

Comment:
typo, but also this function is not doing anything more than `id.split("#")`, because the array is reconstructed.